lib/pipeline.js

```diff
@@ -441,14 +361,17 @@ class PipelineModel extends BaseModel {
                     token
                 })
             ]).then(([existingJobs, openedPRs]) => {
-                const jobList = this._checkPRState(existingJobs, openedPRs);
+                const synced = [];
 
-                return Promise.all([
-                    this._createPRJob(existingJobs, jobList.toCreate),
-                    this._updateJobArchive(jobList.toArchive, true),
-                    this._updateJobArchive(jobList.toUnarchive, false)
-                ]);
-            }));
+                openedPRs.forEach((openedPR) => {
+                    const prNum = openedPR.name.split('-')[1];
+
+                    synced.push(this.syncPR(prNum));
+                });
+
+                return Promise.all(synced)
+                    .then(() => this._archiveClosePRs(existingJobs, openedPRs));
```

Comment:
[ask] why do we need to call this function? It seems that closed PR job will be archived in syncPR function.

Reply:
To archive jobs which are in closed PR.
SyncPR function does not have this feature, thus I add this function.

Reply:
👍